pkg/api/defaults.go

```diff
@@ -77,5 +77,7 @@ func DefaultOperatorFlags() OperatorFlags {
 		"aro.storageaccounts.enabled":              flagTrue,
 		"aro.workaround.enabled":                   flagTrue,
 		"aro.autosizednodes.enable":                flagFalse,
+		"rh.srep.muo.enabled":                      flagTrue,
+		"rh.srep.muo.managed":                      flagTrue,
```

Comment:
This may have been answered in a different PR, but why do these two flags start with `rh.srep` instead of `aro`?

Reply:
Yeah, that's pretty much it. If we're adding new components from not-ARO (e.g. a new controller that deploys a Microsoft thing) then we can prefix it to differentiate it to the SRE. Although one can look at the code to find the origin of things, if a SRE only has the flag name (e.g `rh.srep.muo`, and one is in an escort on govcloud) then the prefix gives a bit of extra information to help identify where it comes from.